pkg/applicationserver/applicationserver.go

```diff
@@ -57,6 +58,8 @@ import (
 	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
+const packageName = "go.thethings.network/lorawan-stack/pkg/applicationserver"
```

Comment:
This looks like a great candidate to put in `observability.go`. Also isn't this more like a tracer namespace?

Another nit; the package name is `go.thethings.network/lorawan-stack/v3/pkg/applicationserver`. I'm not sure though if we need package names; why not namespaces as we use in logging?